app/experimenter/experiments/migrations/0195_nimbusbranch_feature_enabled.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.5 on 2021-11-05 19:12
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("experiments", "0194_auto_20211102_1749"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="nimbusbranch",
+            name="feature_enabled",
+            field=models.BooleanField(default=True),
```

Comment:
In the components that still use this flag, is an empty feature config (i.e. `{}`) equivalent to a non-enabled feature? Or do we need to re-introduce explicit True / False values with some logic in this migration?

Reply:
No, empty value isn't the same as enabled, many experiments have empty in both branches with enabled off/on in each.  Actually I was originally gonna just set everything back to `True` as the default and update any if anyone asked in the admin, but then I remembered we can recover that data from the `published_dto` so I added that to the migration.